bolt/test/AArch64/adrpass.s

```diff
@@ -0,0 +1,42 @@
+# The second and third ADR instructions are non-local to functions
+# and must be replaced with ADRP + ADD by BOLT
+
+# REQUIRES: system-linux
+
+# RUN: %host_cc %cflags %s -o %t.exe -Wl,-q
```

Comment:
I guess you forgot to add llvm-mc before host_cc here?

Reply:
Sorry, removed while playing around and testing :)